demos/tests/run_tests.py

```diff
@@ -219,6 +244,14 @@ def option_to_args(key, value):
                         continue
 
                     for device, dev_arg in device_args.items():
+                        skip = False
+                        for model in test_case_models:
+                            if unsupported_devices and device in unsupported_devices.get(model, []):
+                                print('Test case #{}/{}: Model {} is not supported by device'
```

Comment:
Maybe "not supported" is not good term for logging output here. In some test scenarios models/devices might be excluded from testing, but generally they are supported by device.

Reply:
well, might be "Model {} is suppressed on device" ?

Reply:
Done